src/routes/user.py

```
@@ -78,12 +81,21 @@ async def get_users(
 
+@user_router.get("/executive/users/read")
+async def get_user_summaries(
+    current_user: UserDep,
+    user_service: UserServiceDep,
+) -> Sequence[UserSummaryResponse]:
+    return user_service.get_user_summaries(current_user)
+
+
 @user_router.get("/executive/user/{id}", response_model=UserResponse)
 async def get_user_by_id(
     id: str,
```

Ruff (0.15.5)

[error] Function argument `id` is shadowing a Python builtin

(A002)